pkg/api/v1/defaults.go

```diff
@@ -160,6 +160,18 @@ func SetDefaults_Pod(obj *Pod) {
 			}
 		}
 	}
+	for i := range obj.Spec.InitContainers {
+		if obj.Spec.InitContainers[i].Resources.Limits != nil {
```

Comment:
Can we do this in `SetDefaults_Container`? It seems `SetDefaults_Pod` not necessary :).

BTW, in beta, InitContainers is set after converting; so we're also ignore this for beta, right.